lib/types/storage/Elasticsearch.ts

```diff
@@ -0,0 +1,33 @@
+import { ByteSize, ClusterNodesStats } from "@elastic/elasticsearch/api/types";
+
+export type InfoResult = {
+  type: string;
+  version: string;
+  status?: string;
+  lucene?: string;
+  spaceUsed?: ByteSize;
+  nodes?: ClusterNodesStats;
+};
+
+export type KRequestBody<T> = T & {
+  _kuzzle_info?: {
+    author: string;
+    createdAt: number;
+    updatedAt: number | null;
+    updater: string | null;
+  };
+};
+
+export interface JSONObject {
+  [key: string]: JSONObject | any;
```

Comment:
This union are always resolved to `any` because, any include JSONObject.
Also why not use `JSONObject` type from `kuzzle-sdk` package ?

Reply:
Yes, maybe we should merge SDK JS in this repo to facilitate evolutions or generate a types package from the core and use it in SDK to avoid duplicated types in both packages